### PR TITLE
Fix a bug system ui was displayed incorrectly on remote control

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch fixes a bug system ui was displayed incorrectly on remote control.
When the video requests fullscreen mode with options like navigationUI[1], the
navigation bar would be displayed on the content view. We can hide the
navigation bar by setting the system ui for the content view.

[1] https://tv.naver.com/v/15346940/list/67096

Refs: #560

### CODE REVIEW

File: src/org/triple/banana/remote_control/RemoteControlViewImpl.java [hunk 1]
```diff
@@ -61,8 +62,17 @@ private void setBrightness(float value) {
...
+                | View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY | View.SYSTEM_UI_FLAG_LOW_PROFILE
+                | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION | View.SYSTEM_UI_FLAG_HIDE_NAVIGATION;
+        mDialog.getWindow().getDecorView().setSystemUiVisibility(flags);
+
+        BananaTab tab = org.banana.cake.interfaces.BananaTabManager.get().getActivityTab();
+        if (tab == null || tab.getContext() == null) return;
+        final View contentView = tab.getContentView();
+        if (contentView == null) return;
+        contentView.setSystemUiVisibility(flags);
+        contentView.requestLayout();
```
Comment (romandev) [inline src/org/triple/banana/remote_control/RemoteControlViewImpl.java:75]: Is this really needed?

Comment (jdragonbae) [reply src/org/triple/banana/remote_control/RemoteControlViewImpl.java]: It seems not needed. Thanks for the review.

File: banana_cake/java/org/banana/cake/CakeTab.java [hunk 2]
```diff
@@ -66,4 +67,10 @@ public void exitFullscreen() {
         }
         return new CakeMediaCommandProcessor(MediaSession.fromWebContents(mTab.getWebContents()));
     }
+
+    @Override
+    public View getContentView() {
```
[the remark this answers is not in the document]
Comment (jdragonbae) [reply banana_cake/java/org/banana/cake/CakeTab.java]: Done.

File: banana_cake/interfaces/java/org/banana/cake/interfaces/BananaTab.java [hunk 3]
```diff
@@ -24,4 +25,5 @@ static BananaTab get() {
     void exitFullscreen();
     @NonNull
     BananaMediaCommandProcessor getMediaCommandProcessor();
+    View getContentView();
```
[the remark this answers is not in the document]
Comment (jdragonbae) [reply banana_cake/interfaces/java/org/banana/cake/interfaces/BananaTab.java:30]: Done.

**Pull request merged**